src/common/code_environment.cc

```diff
@@ -79,6 +79,35 @@ int get_process_name(char *buf, int len)
   return 0;
 }
 
+#elif defined(_WIN32)
+
+int get_process_name(char *buf, int len)
+{
+  if (len <= 0) {
+    return -EINVAL;
+  }
+
+  char location[MAX_PATH + 1];
+  int length = GetModuleFileNameA(nullptr, location, sizeof(location) - 1);
```

Comment:
```suggestion
  int length = GetModuleFileNameA(nullptr, location, sizeof(location));
```

see https://docs.microsoft.com/en-us/windows/win32/api/libloaderapi/nf-libloaderapi-getmodulefilenamea